hooks/boot/20-certs_update.rb

```diff
@@ -0,0 +1,36 @@
+# Add options around regenerating certificates
+if module_present?('katello')
```

Comment:
Should this be the `certs` module? Previously it was also applied to the fpc scenario, now only katello.
```suggestion
if module_present?('certs')
```